### PR TITLE
Review for DM-7901: Combine covariance matrices...

### PR DESCRIPTION
**Pull request by jdswinbank**
NB not for merging to master.

### CODE REVIEW

File: policy/HscMapper.paf [hunk 1]
```diff
@@ -159,6 +159,14 @@ exposures: {
         level:       "Skytile"
         tables:      raw_skytile
     }
+    deepCoadd_tempCov: {
+        template:    "deepCoadd/%(filter)s/%(tract)d/%(patch)s/warp-%(filter)s-%(tract)d-%(patch)s-%(visit)d.fits"
```
Comment (jdswinbank) [inline policy/HscMapper.paf]: Isn't this exactly the same as the `tempExp`? Won't they clobber each other?

File: policy/HscMapper.paf [hunk 2]
```diff
@@ -159,6 +159,14 @@ exposures: {
...
+        python:      "lsst.afw.image.ImageD"
+        persistable: "ImageD"
+        storage:     "FitsStorage"
+        level:       "Skytile"
+        tables:      raw_skytile
+    }
     deepCoadd_bgRef: {
```
Comment (jdswinbank) [inline policy/HscMapper.paf]: Be nice to see a bit of white space before and after the block you've added.

File: policy/HscMapper.paf (hunk 1 above)
Comment (AstroVPK) [inline policy/HscMapper.paf]: Agreed and fixed.

File: policy/HscMapper.paf (hunk 2 above)
Comment (AstroVPK) [inline policy/HscMapper.paf]: Added.

**Comment by jdswinbank** [comment]
obsolete.

**Pull request closed without merging**